Error: Use `aria-describedby` to reference a group description. Note: If the checkbox control is not nested in its `label`, wrapping the two elements within another element (such as a `div`) is preferable. Having a shared parent element gives designers more control over the eventual layout of the checkbox group than a `br` does.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group with Accessible Description/2/29f6d078-f0d4-4958-9849-d590e68c244a.html b/output/Checkbox Group with Accessible Description/2/29f6d078-f0d4-4958-9849-d590e68c244a.html
index 88cb5ae..97928b2 100644
--- a/output/Checkbox Group with Accessible Description/2/29f6d078-f0d4-4958-9849-d590e68c244a.html	
+++ b/output/Checkbox Group with Accessible Description/2/29f6d078-f0d4-4958-9849-d590e68c244a.html	
@@ -1,10 +1,16 @@
-<fieldset>
+<fieldset aria-describedby="fruit-desc">
   <legend>What fruits do you like?</legend>
-  <p>You can change this later</p>
-  <input type="checkbox" id="bananas" name="fruits" value="bananas">
-  <label for="bananas">Bananas</label><br>
-  <input type="checkbox" id="oranges" name="fruits" value="oranges">
-  <label for="oranges">Oranges</label><br>
-  <input type="checkbox" id="apples" name="fruits" value="apples">
-  <label for="apples">Apples</label>
+  <p id="fruit-desc">You can change this later</p>
+  <div>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples">
+    <label for="apples">Apples</label>
+  </div>
 </fieldset>
\ No newline at end of file
